feg/protos/s6a_proxy.proto

```diff
@@ -217,9 +217,9 @@ message UpdateLocationAnswer {
 
     message AggregatedMaximumBitrate {
         // Maximum uplink bitrate
-        uint32 max_bandwidth_ul = 1;
+        uint64 max_bandwidth_ul = 1;
         // Maximum downlink bitrate
-        uint32 max_bandwidth_dl = 2;
+        uint64 max_bandwidth_dl = 2;
```

Comment:
did not we add the unit information to cover cases for >4gbps? what is the reason for this change then?

Reply:
I changed the datatypes because I am using etc/magma/subscriberdb.yml to enter default ambr rates. These changes need not to be a part of this PR and I will revert.